### DISCUSSION TITLE
Dependency graphql-java/graphql-java-spring deprecation in favor of spring-project/spring-graphql

### DISCUSSION THREAD

**Discussion by oazmon** [Netflix/dgs-framework#895] [General]
DGS is a wrapper around https://github.com/graphql-java/graphql-java-spring from the GraphQL Java team. Whose repo in [github.com](http://github.com/) has but one sentence in its README.md: "This project is archived in favor of the official [Spring GraphQL](https://github.com/spring-projects/spring-graphql) integration." What is the plan here?

**Comment by srinivasankavitha** [comment]
DGS is not a wrapper around graphql-java-spring. It uses `graphql-java` for processing the graphql query, which is very much actively maintained. `Spring GraphQL` is an alternative to the DGS framework which provides similar functionality and is more recent, while the DGS framework has been in use for the past couple years.
